src/dtos/update-student-profile-info.dto.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  IsArray,
+  IsEmail,
+  IsEnum,
+  IsInt,
+  IsNotEmpty,
+  IsNumber,
+  IsOptional,
+  IsString,
+  Min,
+} from 'class-validator';
+import {
+  ExpectedContractType,
+  ExpectedWorkType,
+  StudentStatus,
+} from '../entities/students-entity';
+
+export class UpdateStudentProfileInfoDto {
+  @IsEmail()
+  @IsNotEmpty()
+  email: string;
+
+  @IsOptional()
+  @IsNumber()
```

Comment:
Tak jak rozmawialiśmy, to jednak ciąg znaków, nie liczba (i tak zostało poprawione w encji). Jako liczba numer nie jest nam do niczego potrzebny, będą problemy z wyświetlaniem i z odczytem, jak ktoś wpisze ze spacjami czy dywizami.
```suggestion
  @IsString()
```